department-of-veterans-affairs/content-build
# Makes CMS ready-check use same host address as build.

## Description

Closes department-of-veterans-affairs/va.gov-cms#12289.

### Before

![Screen Shot 2023-01-24 at 2 02 45 PM](https://user-images.githubusercontent.com/1318579/214384832-ddbb9224-836f-48b1-abda-fe41e9735d08.png)


### After

![Screen Shot 2023-01-24 at 1 56 39 PM](https://user-images.githubusercontent.com/1318579/214383579-cba707ba-c8a2-48a2-bd7b-7cbdd94125e5.png)


## Testing done & Screenshots

## QA Steps
Verify that on CI builds that the CMS-ready-check hostname matches that used in the build.
